dialogs/energy_spectrum.py

```diff
@@ -193,6 +201,40 @@ def __init__(self, parent: BaseTab, spectrum_type: str = _MESU,
         self.__update_eff_files()
         self.exec_()
 
+    def remove_external_file(self):
```

Comment:
Isn't the point of this function to remove all selected external files? If so, it should be renamed more descriptively (`remove_selected_external_files` for example).

Reply:
Yes, you are right.